### PR TITLE
Venv configuration

### PR DESCRIPTION
**Pull request by renjingxiao**
Hi Andrea/Paolo,

I added a myenv folder and a blank requirements.txt. Please correct me if I've made any mistakes.

Regards,
Jean

### CODE REVIEW

**Comment by andreaTP** [comment]
Hi @renjingxiao and thanks a lot for this PR!

As far as I understand this configuration should be sufficient to:
- commit only `requirements.txt`
- add `.env` to the `.gitignore` file
- remove the `myenv` folder
- add instructions on how to replicate the environment, I expect something along those lines: `virtualenv .env && source .env/bin/activate && pip install -r requirements.txt`

Ideally, but not required, using the virtualenv in CI should guarantee everything is properly automated.

File: README.md [hunk 1]
```diff
@@ -14,6 +14,18 @@ yarn install
...
 REGISTRY_ENDPOINT="http://localhost:8080" yarn dev
 ```
+## Replicate the environment
+
+```sh
+# create a new environment
+python3.10 -m venv .env
+# activate it
+source .env/bin/activate
+# or this if on windows
```
Comment (andreaTP) [inline README.md:24]: I would keep the windows instructions in the comment as ppl tend to copy-paste instructions directly in the terminal ...

**Review by andreaTP — COMMENTED** [review]
2 minor comments and LGTM, thanks!

**Comment by renjingxiao** [comment]
Thank you Andrea! That makes more sense and I have changed them.
I have also moved the environment section before the "Running Locally" section and would like to know if it's better.
Thanks again for your time and patience!

**Review by andreaTP — APPROVED** [review]
Sorry for the delay, LGTM! Thanks a lot!

**Comment by andreaTP** [comment]
Hi all and sorry for the late check, I figured out that the current `requirements.txt` is including some non-portable library like: https://github.com/Apicurio/backstage/blob/0e907e279315f97ce766075b1c09119be7fe6c65/requirements.txt#L80

and is probably installing quite a number of dependencies that are not strictly needed.
Please note that I tested in CI on a `macos-latest` runner and locally on my Mac and I cannot reproduce the original issue, unfortunately.

**Pull request merged**